processor/src/main/kotlin/com/livefront/sealedenum/internal/processor/SealedEnumProcessor.kt

```diff
@@ -237,7 +240,9 @@ internal class SealedEnumProcessor : AbstractProcessor() {
                 kmClass.isSealed -> createSealedClassNode(kmClass)
                 else -> throw NonObjectSealedSubclassException(element)
             }
-            else -> throw InvalidSubclassVisibilityException(element)
+            else -> throw InvalidSubclassVisibilityException(
```

Comment:
Minor, but this formatting change looks unnecessary.

Reply:
Updated!